app/_icons/modules.tsx

```diff
@@ -0,0 +1,61 @@
+import { createIcon } from '@chakra-ui/icons';
+
+export const Systems = createIcon({
+  displayName: 'Systems',
+  viewBox: '0 0 21 21',
+  d: 'M0.5 10.4C0.5 4.87999 4.98 0.399994 10.5 0.399994C16.02 0.399994 20.5 4.87999 20.5 10.4C20.5 15.92 16.02 20.4 10.5 20.4C4.98 20.4 0.5 15.92 0.5 10.4ZM10.5 16.4C13.81 16.4 16.5 13.71 16.5 10.4C16.5 7.08999 13.81 4.39999 10.5 4.39999C7.19 4.39999 4.5 7.08999 4.5 10.4C4.5 13.71 7.19 16.4 10.5 16.4Z',
+});
```

Comment:
Is there no other way to just import these fonts or adopt an icon library that is extensive enough to meet out needs. In the design I had used fonts from material UI or something. This is an important decision, getting these definitions every time we want to use a font can be an overhead I would like to avoid.

Reply:
I figured these icons were WIP and would be replaced at some point. I've used this kind of strategy for custom icons before and it works well. But, if we'd rather pull in material icons or another library, that's cool. I'd vote for that in a separate ticket/PR tho. thoughts?

Reply:
How do you get that icon value? If others in the project have to use a new icon, how do we generate that?

Reply:
From the figma file. Select the vector node > Copy as > svg

Reply:
Yeah, so the process of getting that is cumbersome as we decided not to build every view in Figma and to use code as the first attempt to show the UX ideas. For this to work, we will need to get this icon code somehow, if not from Figma. Any other easy ways to get this?

Reply:
I haven't shopped the depths of 3rd party icon libs in a while, not sure which ones would support the ED/space specifics we might want. But, I figure we'll have to draw out any custom one in some tool (I can't write svg paths by hand, lol).

Reply:
Opened a new issue for this with some recommendations, not important to handle in this PR though - https://github.com/ed-pilots-network/frontend/issues/31